.editorconfig

```diff
@@ -1,36 +1,34 @@
+# top-most EditorConfig file.
 root = true
 
+# Sensible defaults for all file types.
 [*]
 charset = utf-8
 end_of_line = lf
-indent_style = space
-indent_size = 2
 insert_final_newline = true
-max_line_length = 80
 trim_trailing_whitespace = true
 
-[*.lua]
-max_line_length = 120
-
-[*.sh]
-shell_variant      = posix # like -ln=posix
-binary_next_line   = true  # like -bn
-switch_case_indent = true  # like -ci
+[[bash]]
+indent_style = space
+indent_size = 4
+language_dialect = bash
 
 [*.bash]
-shell_variant      = bash  # like -ln=posix
-binary_next_line   = true  # like -bn
-switch_case_indent = true  # like -ci
+indent_style = space
+indent_size = 4
+language_dialect = bash
 
-[**/git/config**]
-indent_style = tab
+[[shell]]
+indent_style = space
 indent_size = 4
+language_dialect = posix
 
+[*.sh]
+indent_style = space
+indent_size = 4
+language_dialect = posix
```

Comment:
Can the duplication be removed?

Also what is [[bash]] targeting? Is this shebangs?

Reply:
#557